src/components/BalanceCard/index.tsx

```diff
@@ -6,7 +6,7 @@ import { FilecoinNumber } from '@glif/filecoin-number'
 import { ButtonV2 } from '../Button/V2'
 import { OutlineBox } from '../Layout'
 import { FILECOIN_NUMBER_PROPTYPE } from '../../customPropTypes'
-import makeFriendlyBalance from '../../utils/makeFriendlyBalance'
+import { makeFriendlyBalance } from '../../utils'
```

Comment:
I think we should leave the more specific import here, this way we don't _necessarily_ bring in the entire `utils` directory when we import `BalanceCard`